middleware/src/main/java/br/ufrn/dimap/middleware/identification/lookup/Lookup.java

```diff
@@ -22,8 +23,12 @@
 	 * 				reference.
 	 * @param aor	The absolute object reference that must be binded with the 
 	 * 				given name (propertied).
+	 * @param host Server connection host.
+	 * @param port Server connection port.
+	 * @throws RemoteError
 	 */
-	public void bind(String name, AbsoluteObjectReference aor) throws RemoteError;
+	public void bind(String name, Object remoteObject, String host, int port) 
```

Comment:
We will provide (probably today) a common interface for the Invoker, so that instead of "Object remoteObject", you can write "Invoker remoteObject", or something like that. Apart from that, everything is fine for me.